lib/tasks/publications_export.rake

```diff
@@ -1,14 +1,15 @@
 namespace :publications do
   desc "Export all publications for a document collection to a CSV file."
-  task :export_for_document_collection, [:organisation_slug] => :environment do |_, args|
+  task :export_for_document_collection, [:organisation_slug, :documents_collection_slug] => :environment do |_, args|
```

Comment:
It would be good to document the rake tasks (and how to run them) in the PR description. I wasn't really expecting to see a documents_collection_slug parameter.